src/index.js

```diff
@@ -0,0 +1,70 @@
+import './style.css';
+import reload from './assets/reload.svg';
+import entericonss from './assets/EnterIcon.svg';
+import data from './dataArray.js';
+import Trash from './assets/TrashBasket.svg';
+
+class TodoList {
+  constructor() {
+    this.todoList = document.getElementById('todo_list');
+    this.todoDescription = document.getElementById('description');
+    this.clearAllButton = document.getElementById('clear_all_button');
+    this.reloadButton = document.getElementById('reload');
+    this.dataArray = data;
+
+    this.displayItems();
+
+    this.reload();
+  }
+
+      displayItems = () => {
+        this.clearList();
+        for (let i = 0; i < this.dataArray.length; i += 1) {
+          const todo = this.dataArray[i];
+          const todoItem = document.createElement('li');
+          todoItem.classList.add('todo_item');
+
+          const check = document.createElement('input');
+          check.type = 'checkbox';
+          check.id = this.dataArray[i].index;
+          check.classList.add('checkbox');
+          if (todo.completed === true) {
+            check.checked = true;
+          }
+          todoItem.appendChild(check);
+
+          const todoItemDescription = document.createElement('span');
+          todoItemDescription.classList.add('description');
+          if (todo.completed === true) {
+            todoItemDescription.classList.add('completed');
+          }
+          todoItemDescription.textContent = `${todo.description}`;
+          todoItem.appendChild(todoItemDescription);
+
+          const trashIcon = document.createElement('img');
+          trashIcon.src = Trash;
+          trashIcon.classList.add('trash');
+          todoItem.appendChild(trashIcon);
+          trashIcon.id = this.dataArray[i].index;
+
+          this.todoList.appendChild(todoItem);
+        }
+        const loadImg = document.querySelector('.reload');
+        const entericons = document.querySelector('.enter_icon');
+        loadImg.src = reload;
+        entericons.src = entericonss;
+      }
+
+      clearList = () => {
+        this.todoList.innerHTML = '';
```

Comment:
- [ ] This implementation is not meeting the requirement yet and I think you may have to check your iteration again because no task gets listed upon a `load` event. See the rendering below:
![image](https://github.com/tatouroberto/T0-Do-List/assets/99715640/1102ec4b-9943-4e37-a625-8c56f9b9e3ce)

Good job so far!